lib/screens/forum/page.dart

```diff
@@ -5,9 +5,9 @@ import 'package:Solon/services/forum_connect.dart';
 import 'package:Solon/util/app_localizations.dart';
 import 'package:Solon/util/forum_util.dart';
 import 'package:Solon/widgets/bars/page_app_bar.dart';
+import 'package:Solon/widgets/cards/comment_card.dart';
 import 'package:flutter/material.dart';
 import 'package:translator/translator.dart';
```

Comment:
we dont need this package but how did this come back ? i thought we rid of it last month

Reply:
which package are you talking about

Reply:
import 'package:translator/translator.dart'; in lib/screens/forum/page.dart; this was for google translate back in august if i remember correctly